EDA.qmd

```diff
@@ -290,13 +266,6 @@ If you've encountered unusual values in your dataset, and simply want to move on
       mutate(y = if_else(y < 3 | y > 20, NA, y))
     ```
 
-`if_else()` has three arguments.
```

Comment:
IMO this is the biggest benefit of the TRs — spotting when we're moved stuff so we accidentally use it before we teach it.

Reply:
Agreed, those have been super helpful!